cli/app.go

```diff
@@ -215,13 +217,13 @@ var app = &cli.App{
 				},
 				{
 					Name:      "delete",
```

Comment:
Wanted to get this in before the release so that we could finally fully remove DeleteTabularDataByFilter from the API without any stable RDK or app dependencies on it.

Don't want to split this into delete-binary since external documentation relies on this, so instead staging it this way first: update CLI -> remove deprecated API -> coordinate following CLI release + documentation update to `delete-binary` and `delete-tabular`

Reply:
Thanks for explaining!